README.md

```diff
@@ -93,3 +93,94 @@ extension BannerView: BannerRendering {
 }
 ```
 
+## 🛰 Interactor
+
+> **Interactor**
+> (*noun*)
+> 
+> Component answering the question: What should I do in reaction to user action. 
+
+### Definitions
+
+### TL;DR
+
+Interactor is a final class that ViewController owns, which performs actions in reaction to user input - gestures, text input, device movement, geographical location change etc.
```

Comment:
Is it owned only by `ViewController`? 🤔

Reply:
Yes, every view can own it's own Interactor but for simple VC this is not necessary and one Interactor can handle couple of them.